source/Introduction/How-SpaceROS-Differs.rst

```diff
@@ -1,22 +1,118 @@
 How Space ROS Differs from ROS 2
 ================================
 
-TODO:
-
-Provide a high-level overview of the various features of Space ROS and describe how Space ROS is different from ROS 2.
-
-Some example topics:
-
-* Space ROS website (and domain names)
-* Space ROS documentation site (this site)
-* Space ROS github organization
-* Additional static analysis tools and SARIF output (IKOS, Cobra, MC/DC testing, etc.)
-* The Space ROS requirements tools and processes
-* Space ROS Quality Level(s)
-* Earthly for local developer builds and CI
-* C++ PMR memory allocator
-* The Eventing & Telemetry Subsystem
-* Space-related demo applications
-* Space-related simulation assets
-* RTEMS
-* cFS/ROS 2 Bridge
+To date, work has focused on establishing the foundation for Space ROS, including builds and infrastructure, code analysis tools, requirements tools and processes, and space-specific functionality.
+These efforts extend the foundation provided by ROS 2.
+The following sections list the differences between Space ROS and the standard ROS 2 distributions.
+
+Builds and Infrastructure
+-------------------------
+..
+  Space ROS builds include a subset of ROS 2 packages.
+
+There are several new infrastructure elements for Space ROS, including  information resources and regular builds provided to the Space ROS community.
+
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Item
+     - Description
+   * - Website
+     - The `Space ROS website <https://space-ros.github.io/spaceros.org/>`_ has high-level information about Space ROS as well as a searchable set of related articles and videos.
+   * - Documentation site
+     - The Space ROS documentation site (this site) contains information about Space ROS, without duplicating the standard ROS 2 documentation.
+       For standard ROS 2 topics, one can refer to the `ROS 2 Humble documentation <https://docs.ros.org/en/humble/index.html>`_.
+   * - Github organization
+     - There is a `github organization <https://github.com/space-ros>`_ for all of the Space ROS projects.
+   * - Docker images
+     - Pre-built Docker images are available from the `OSRF Space ROS page <https://hub.docker.com/r/osrf/space-ros>`_ on Docker Hub.
+   * - Earthly builds
+     - Space ROS builds use the `Earthly build tool <https://earthly.dev/>`_.
```

Comment:
```suggestion
     - Space ROS builds use the `Earthly CI/CD tool <https://earthly.dev/>`_.
```

"build tool" has a very precise definition within ROS (see http://design.ros2.org/articles/build_tool.html) which Earthly does not meet so I propose rephrasing this slightly.

Reply:
Fixed.